### PR TITLE
UPDATE: support arm64

### PR DESCRIPTION
**Pull request by Segelzwerg**
Also updates the docker image test process to use the same steps in order to verify build before publish.

### CODE REVIEW

File: .github/workflows/docker-image.yml [hunk 1]
```diff
@@ -8,10 +8,25 @@ on:
...
+        uses: docker/setup-buildx-action@v1
+      - name: Build
+        id: docker_build
+        uses: docker/build-push-action@v2
+        with:
+          push: false
+          file: ./Dockerfile
+          platforms: linux/${{ matrix.arch }}
+          tags: |
+            whist-team/whist-server:${{ matrix.arch }}-test
```
Comment (iTitus) [inline .github/workflows/docker-image.yml:32]: wouldnt it make more sense to use the commit hash here?

**Review by iTitus — CHANGES_REQUESTED** [review]

File: .github/workflows/docker-image.yml (hunk 1 above)
Comment (Segelzwerg) [reply .github/workflows/docker-image.yml:32]: Yes, but the image is obsolete after the run

File: .github/workflows/docker-image.yml [hunk 2]
```diff
@@ -8,10 +8,25 @@ on:
...
+        uses: docker/setup-buildx-action@v1
+      - name: Build
+        id: docker_build
+        uses: docker/build-push-action@v2
+        with:
+          push: false
+          file: ./Dockerfile
+          platforms: linux/${{ matrix.arch }}
+          tags: |
+            whist-team/whist-server:${{ matrix.arch }}-${{ github.event.pull_request.head.sha }}
```
Comment (iTitus) [inline .github/workflows/docker-image.yml:32]: will this actually work with normal pushes?

Comment (Segelzwerg) [reply .github/workflows/docker-image.yml:32]: Ah, good, I guess not. I'll have look later

Comment (Segelzwerg) [reply .github/workflows/docker-image.yml]: https://github.community/t/add-short-sha-to-github-context/16418/12 let's keep 'test'

**Review by iTitus — APPROVED** [review]

**Pull request merged**